main/src/cgeo/geocaching/ui/CacheDetailsCreator.java

```diff
@@ -227,4 +230,65 @@ public TextView addHiddenDate(@NonNull final Geocache cache) {
         view.setId(R.id.date);
         return view;
     }
+
+    public void addLatestLogs(final Geocache cache) {
+        Context context = parentView.getContext();
+
+        final RelativeLayout layout = (RelativeLayout) activity.getLayoutInflater().inflate(R.layout.cache_information_item, null, false);
+        final TextView nameView = layout.findViewById(R.id.name);
+        nameView.setText(res.getString(R.string.cache_latest_logs));
+        final LinearLayout markers = layout.findViewById(R.id.linearlayout);
+
+        int smileySize = 48;
```

Comment:
This will be a size in px, correct? That would lead to problems on screens with different densities.

Instead of using an absolute px value, would using one of the constants defined in DisplayUtils (and converting dp to px) help?

Reply:
What I wanted to achieve is to make them as large as a line of text, not some explicit size. But stuff like parent.getHeight() didn't work and then I just went for a hardcoded size to get something.
If you have some idea how to properly grow them to the right height I'll use that (MATCH_PARENT works for height, but then width is wrong with both MATCH_PARENT and WRAP_CONTENT), otherwise I'll try to find some other constant that can be used.

Reply:
Can you use some of the scaling options for images, e. g. fit_center, and set height for surrounding LinearLayout to match_parent?

Or, if that does not work, you could use some "sp" value to take font scaling and different densities into account. 

Hard-coded px will only work for specific combinations of font scaling (see Android's system settings) and densities.

Reply:
I couldn't figure out something with match_parent. For some reason I can't get Layout Inspector to work with c:geo, so trying to figure out what's going wrong here is a pain in the ...

I've now switched to using the text size as a base.